Dockerfile

```diff
@@ -0,0 +1,37 @@
+# BASE IMAGE
+FROM python:latest as base
+
+# UPDATE IMAGE/UPGRADE PIP/INSTALL POETRY
+RUN apt-get update && pip install --upgrade pip && pip install poetry --no-cache-dir
+
+# COPY APP CODE
+WORKDIR /app
+COPY . .
+
+# INSTALL DEPENDENCIES INC. GUNICORN
+RUN poetry install --only main 
+
+# PROD CONTAINER==========================================================================
+FROM base as production
+
+# PROD - EXPOSE PORT 
+ENV WEBAPP_PORT=8000
+EXPOSE ${WEBAPP_PORT}
+
+# PROD - ENTRYPOINT/CMD
+ENTRYPOINT ["poetry", "run", "gunicorn", "--bind", "0.0.0.0", "todo_app.app:create_app()"]
+
+# ALSO WORKS
+# CMD ["poetry", "run", "gunicorn", "--bind", "0.0.0.0", "todo_app.app:create_app()"]
```

Comment:
Happy to leave these here for your reference if useful, but if this was a real project I'd strongly push for getting rid of them, because inevitably these will end up out of date when someone changes the currently used entrypoint, and then someone else will try to rely on these comments!